lib/display.ts

```diff
@@ -45,10 +45,18 @@ function displayIssues(
   const depCount = pkgCount == 1 ? '1 dependency' : `${pkgCount} dependencies`;
   const issuesCount =
     issues.length == 1 ? '1 issue' : `${issues.length} issues`;
-  result.push(chalk.whiteBright('Issues'));
-  for (const { pkgName, pkgVersion, issueId } of issues) {
-    const { title, severity } = issuesData[issueId];
 
+  const hasPkgsAndIssues = pkgCount > 0 && issues.length > 0;
+  if (hasPkgsAndIssues) {
+    result.push(chalk.whiteBright('Issues'));
+  }
+
+  for (const {
```

Comment:
Since this for-loop is coupled (at least in behavior), to the if-statement at line 50, I might want to insert the for-loop inside the if-block. Usually I'm not a big fan of if-blocks over more line than necessary, but it might be a visual thing for the reader that it relates to each other. Alt move "Issues"-header and the for-loop to it's own function, and call that function from the if-hasPkgsAndIssues block.

Reply:
<img width="1239" alt="Screen Shot 2021-09-13 at 14 52 36" src="https://user-images.githubusercontent.com/40601533/133078819-ecfff249-92ad-4011-86d3-115bf126151a.png">

Taking bits from https://github.com/snyk/snyk-cpp-plugin/pull/22

that one will be improved with another pr

Reply:
Trying to introduce small changes to avoid massive code changes